src/EditorFeatures/Test/CodeFixes/ExtensionOrderingTests.cs

```diff
@@ -88,12 +88,12 @@ public void TestNoCyclesInRefactoringProviders()
             // ExtensionOrderer.Order() will not throw even if cycle is detected. However, it will
             // break the cycle and the resulting order will end up being unpredictable.
             var actualOrder = ExtensionOrderer.Order(csharpProviders).ToArray();
-            Assert.Equal(9, actualOrder.Length);
+            Assert.True(actualOrder.Length > 0);
```

Comment:
A test that says "you have exactly this many refactorings" is not a useful test IMO.  You only ever need to work with it just to change it to update it to the right count.